projects/1.20/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -0,0 +1,580 @@
+	"item.gobber2.gobber2_armor_nether.line5": "靴子：无",
+	"item.gobber2.gobber2_armor_nether.line6": "全套加成：免疫中毒和凋零",
+	"item.gobber2.gobber2_helmet_nether": "下界头盔",
+	"item.gobber2.gobber2_chestplate_nether": "下界胸甲",
+	"item.gobber2.gobber2_leggings_nether": "下界护腿",
+	"item.gobber2.gobber2_boots_nether": "下界靴子",
+
+	"item.gobber2.gobber2_armor_end.line1": "头盔：快速自动补食、饱和、水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_end.line1b": "头盔：水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_end.line2": "胸甲：抗火和击退抗性",
+	"item.gobber2.gobber2_armor_end.line3": "护腿：免疫摔落伤害",
+	"item.gobber2.gobber2_armor_end.line4": "靴子：无",
+	"item.gobber2.gobber2_armor_end.line5": "全套加成：免疫负面效果",
+	"item.gobber2.gobber2_helmet_end": "末地头盔",
+	"item.gobber2.gobber2_chestplate_end": "末地胸甲",
+	"item.gobber2.gobber2_leggings_end": "末地护腿",
+	"item.gobber2.gobber2_boots_end": "末地靴子",
+
+	"item.gobber2.gobber2_armor_dragon.line1": "头盔：快速自动补食、饱和、水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_dragon.line1b": "头盔：水下呼吸和伤害吸收",
+	"item.gobber2.gobber2_armor_dragon.line2": "胸甲：抗火和击退抗性",
+	"item.gobber2.gobber2_armor_dragon.line3": "护腿：免疫摔落伤害",
+	"item.gobber2.gobber2_armor_dragon.line4": "靴子：无",
+	"item.gobber2.gobber2_armor_dragon.line5": "全套加成：创造模式飞行、免疫负面效果",
+	"item.gobber2.gobber2_armor_dragon.line6": "全套加成：免疫负面效果",
+	"item.gobber2.gobber2_helmet_dragon": "龙之头盔",
+	"item.gobber2.gobber2_chestplate_dragon": "龙之胸甲",
+	"item.gobber2.gobber2_leggings_dragon": "龙之护腿",
+	"item.gobber2.gobber2_boots_dragon": "龙之靴子",
+
+	"item.gobber2.gobber2_links_end": "末地盔甲锁链",
+
+	"item.gobber2.sneak_right_click": "潜行右键切换开/关",
+	"item.gobber2.works_in_enderchest": "在末影箱中生效",
+	"item.gobber2.works_in_inventory": "在物品栏中生效",
+	"item.gobber2.works_in_main_hand": "在主手时生效",
+	"item.gobber2.works_in_off_hand": "在副手时生效",
+	"item.gobber2.right_click_use": "右键使用",
+
+	"item.gobber2.gobber2_ring.cooldown": "冷却时间（刻）: %s",
+	"item.gobber2.gobber2_ring": "戒指",
+	"item.gobber2.gobber2_ring_nether": "下界戒指",
+	"item.gobber2.gobber2_ring_end": "末地戒指",
+
+	"item.gobber2.gobber2_ring_above": "乾坤戒指",
+	"item.gobber2.gobber2_ring_above.line1": "将玩家传送到正上方最高处的安全方块上",
+	"item.gobber2.gobber2_ring_above.line2": "右键使用",
+	"item.gobber2.gobber2_ring_above.line3": "潜行右键使用时将玩家传送到正下方最低处的安全方块上",
+	"item.gobber2.gobber2_ring_above.line4": "请注意，向下传送时，您可能会被传送到非常底层的位置！",
+	"item.gobber2.gobber2_ring_above.line5": "很抱歉，这枚戒指仅能在主世界使用",
+
+	"item.gobber2.gobber2_ring_acceleration": "加速戒指",
+	"item.gobber2.gobber2_ring_acceleration.line1": "当玩家在空中时提升速度",
+	"item.gobber2.gobber2_ring_acceleration.line2": "长按右键，然后起跳",
+	"item.gobber2.gobber2_ring_acceleration.line3": "配置的速度值过高",
+
+	"item.gobber2.gobber2_ring_airwalking": "踏空戒指",
+	"item.gobber2.gobber2_ring_airwalking.line1": "提供空中行走的能力",
+	"item.gobber2.gobber2_ring_airwalking.line2": "右键一个空气方块放置戈伯玻璃",
+
+	"item.gobber2.gobber2_ring_ascent": "升空戒指",
+	"item.gobber2.gobber2_ring_ascent.line1": "使玩家在空中飘浮3分钟",
+	"item.gobber2.gobber2_ring_ascent.line2": "在副手中右键使用, 按潜行移除效果",
```

```suggestion
	"item.gobber2.gobber2_ring_ascent.line2": "在副手中右击以使用，潜行以移除效果",
```

标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。